src/modules/blockchain/implementation/gnosis/gnosis-service.js

```diff
@@ -19,26 +19,34 @@ class GnosisService extends Web3Service {
     }
 
     async getGasPrice() {
+        let gasPrice;
         try {
             const response = await axios.get(this.config.gasPriceOracleLink);
-            let gasPrice;
-            if (this.config.name.split(':')[1] === '100') {
+            if (response?.data?.average) {
+                // Returnts gwei
+                gasPrice = Number(response.data.average);
+                this.logger.debug(`Gas price from Gnosis oracle link: ${gasPrice} gwei`);
+            } else if (response?.data?.result) {
```

Comment:
Doesn't matter too much, but I would put this on the first place just for the sake of clarity as this is the default option for Gnosis Mainnet (gnosisscan)